src/components/MerchantsPage/MerchantCard.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import styles from './styles.module.scss';
+import { SellerTwo } from '../../utilities/api';
+import temp from './temp.png';
+
+export interface Props {
+  storeInfo?: SellerTwo;
+}
+
+const MerchantCard = ({ storeInfo }: Props) => {
+  const storeName = storeInfo?.seller_id
```

Comment:
I wouldn't want to rely on the URL to be the actual store name. eg) what if they have a char that doesn't work in URLs in their name such as Café China

Or even if there are multiple 99-favor-tastes, so we have to do 99-favor-taste-1 in the URL, even though it's called 99-favor-taste